titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobAssertions.java

```diff
@@ -56,6 +63,20 @@ public boolean isValidIamRole(String iamRole) {
         return !StringExt.safeTrim(iamRole).isEmpty();
     }
 
+    public boolean isEnvironmentNotTooLarge(Map<String, String> environment) {
```

Comment:
Rename method/constant names to indicate this is for environment variables and not the environment.

Reply:
Done